docs/application/dotnet/guides/alarm/alarm-cover.md

```diff
@@ -1,11 +1,6 @@
 Alarms and Scheduling
 =====================
```

Comment:
Please change to "#  Alarms and Scheduling" and delete "=====================".

I won't leave the same comment  about all the same cases. Please fix the issue.